### PR TITLE
Create CODE-OF-CONDUCT.md

### PR DESCRIPTION
**Pull request by Trueblueaddie**
This is a requested document for all repos. It follows a standard format.

### CODE REVIEW

**Review by coot — APPROVED** [review]
LGTM

File: CODE-OF-CONDUCT.md
Comment (nfrisby) [inline CODE-OF-CONDUCT.md:14]: This list reminds me of @JaredCorduan ❤️

File: CODE-OF-CONDUCT.md
Comment (nfrisby) [inline CODE-OF-CONDUCT.md:36]: I appreciate that we're in the early days of rolling out a carefully-considered IOG-wide approach to Open Source. But as a project maintainer of this repository, I'll feel unprepared for this important responsibility until I get some information/explanations about IOG's more detailed intentions/SOPs/etc regarding eg what is "appropriate and fair corrective action", what evidence do I need before acting, etc.

Comment (nfrisby) [reply CODE-OF-CONDUCT.md:36]: Hmm. Or maybe I'm not actually a _project maintainer_ in this context, and my only responsibility as a "repo owner" would be to contact `code-of-conduct@` as discussed in `## Enforcement` below?

Comment (nfrisby) [reply CODE-OF-CONDUCT.md:36]: Would appreciate any clarification you could provide for the questions I raised in this thread, @Trueblueaddie . Thanks!

File: CODE-OF-CONDUCT.md
Comment (nfrisby) [inline CODE-OF-CONDUCT.md:70]: Why 1.4 and not today's 2.1?

**Review by nfrisby — CHANGES_REQUESTED** [review]
I asked some clarifying questions that I would prefer to understand better before merging this important file. Thanks!

File: CODE-OF-CONDUCT.md
Comment (JaredCorduan) [reply CODE-OF-CONDUCT.md:14]: Thank you @nfrisby, that means a lot to me.

File: CODE-OF-CONDUCT.md
Comment (Trueblueaddie) [reply CODE-OF-CONDUCT.md]: Good point @nfrisby

File: CODE-OF-CONDUCT.md
Comment (Trueblueaddie) [reply CODE-OF-CONDUCT.md:36]: I think the implementation of a CoC, as a first step, is an indication of IOG's commitment to creating a healthy open source community. As far as a SOP goes, there are a few conversations going on with legal and HR - nothing definitive yet. Training is also being discussed to support this new SOP.  

The definition of roles and responsibilities is a critical point. In the documentation request, is an ask for the identification of the core maintainers typically listed in the README.md. It would be good to have an IOG standard for the definition of  a  "repo owner", "project maintainer", "core maintainer", etc.

File: CODE-OF-CONDUCT.md
Comment (Trueblueaddie) [reply CODE-OF-CONDUCT.md:70]: I pulled 1.4 from another IOG repo. 2.1 looks good too

File: CODE-OF-CONDUCT.md
Comment (dnadales) [reply CODE-OF-CONDUCT.md:14]: Absolutely!

File: CODE-OF-CONDUCT.md
Comment (JaredCorduan) [inline CODE-OF-CONDUCT.md:7]:
```suggestion
our community a harassment-free experience for everyone, regardless of but not limited to age, body
```

**Comment by michaelpj** [comment]
Again, I would suggest just linking to the one in the handbook, which has been updated recently: https://github.com/input-output-hk/cardano-engineering-handbook/blob/main/CODE-OF-CONDUCT.md

**Comment by JaredCorduan** [comment]
> Again, I would suggest just linking to the one in the handbook, which has been updated recently: https://github.com/input-output-hk/cardano-engineering-handbook/blob/main/CODE-OF-CONDUCT.md

@michaelpj should we wait for the handbook to become public, or just link to it now?

**Comment by michaelpj** [comment]
It's just recently public, Jared!

**Comment by JaredCorduan** [comment]
> It's just recently public, Jared!

I noticed that immediately after posting, and tried to delete my post before it would bother you :laughing: . congrats!! :tada:

**Review by coot — CHANGES_REQUESTED** [review]
I second linking the [Cardano Engineering Handbook](https://github.com/input-output-hk/cardano-engineering-handbook/blob/main/CODE-OF-CONDUCT.md).

**Pull request closed without merging**
